de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/customer_revenue_credit_limit.sql

```diff
@@ -0,0 +1,134 @@
+DROP FUNCTION IF EXISTS report.customer_revenue_credit_limit 
+( 
+	IN Date date,
+	IN C_BPartner_ID numeric
+);
+CREATE OR REPLACE FUNCTION report.customer_revenue_credit_limit 
+( 
+	IN Date date,
+	IN C_BPartner_ID numeric
+)
+RETURNS TABLE
+(
+	BP_Value character varying(40),
+	BP_Name1 character varying(60), 
+	BP_Name2 character varying(60), 
+	Address1 character varying(100), 
+	Postal character varying(10), 
+	City character varying(60),
+	CountryName character varying(60),
+	Responsible_Sales_Person character varying,
+	Customer_Group character varying,
+	Revenue_3_Months numeric,
+	Revenue_2_Months numeric,
+	Revenue_last_Month numeric,
+	InvoiceSchedule character varying(60),
+	PaymentTerm character varying(60),
+	Paymentrule character varying(60),
+	Creditlimit_Insurance numeric,
+	Creditlimit_Management numeric,
+	isActive character(1),
+	Created date
+)
+AS $$
+
+SELECT  
+	bp.Value AS BP_Value,
+	bp.Name AS BP_Name1,
+	bp.Name2 AS BP_Name2,	
+	l.Address1,
+	l.Postal,
+	l.City,
+	c.Name AS CountryName,
+	bp.salesresponsible AS Responsible_Sales_Person,
+	refl_sg.name AS Customer_Group,
+	
+		SUM( CASE WHEN 	
+			fa.DateAcct::date >= ((date_trunc('month', $1) - interval '3 month')::date)
```

Comment:
please use parameter names instead of $n

Reply:
done